components/action/UnCommentaireAction.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import IconComponent, { IconName } from 'components/ui/IconComponent'
+import { Commentaire } from 'interfaces/action'
+import { formatDayHourDate } from 'utils/date'
+
+interface UnCommentaireActionProps {
+  commentaire: Commentaire
+  idConseiller: string | undefined
+}
+
+export function UnCommentaireAction({
+  commentaire,
+  idConseiller,
+}: UnCommentaireActionProps) {
+  return (
+    <>
+      <div className='flex items-center pb-2'>
+        {commentaire.createur.id !== idConseiller && (
+          <IconComponent
+            name={IconName.People}
+            aria-hidden={true}
+            focusable={false}
+            className='w-4 h-4 mr-2 inline'
+          />
+        )}
+        <span className='text-base-bold mr-2 capitalize'>
```

Comment:
le span tue le span 🥲  les span et les div n'ont pas de valeur sémantique, et servent pour le style.
Dans ce cas, on a une balise avec un sens sémantique, du texte à lire, ce serait ici l'équivalent d'un dt si on reste dans une description list (je me pose la pertinence d'un dl ici encore à vrai dire).
```suggestion
        <dt className='text-base-bold mr-2 capitalize'>
         {commentaire.createur.id === idConseiller
            ? 'vous'
            : `${commentaire.createur.prenom} ${commentaire.createur.nom}`}
            <span className='mr-2'>·</span>
        <span>{formatDayHourDate(commentaire.date)}</span>
        <dt>
```
en regroupant toute la ligne dans une balise donc, et styliser à la convenance chaque partie dans des span